docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_4.md

```diff
@@ -87,7 +86,7 @@ To enable quick access to the filtered data without having to re-run all the met
 
 !!! tip "Tag Names"
 
-    While you can use any tag name you see fit, use **DGE-FILTER-DATA** as used in this lesson to match your screen with the lesson screenshots.
+    While you can use any tag name you see fit, we recommend using **DGE-FILTER-DATA**, used in this lesson to match your screen with the lesson screenshots.
```

Comment:
suggestion: "You can use any tag name. In this lesson and in the screenshots, we use **DGE-FILTER-DATA**."